src/components/tooltip/Tooltip.js

```diff
@@ -243,7 +243,7 @@ class Tooltip extends React.Component<Props, State> {
 
         const bodyEl = bodyElement instanceof HTMLElement ? bodyElement : document.body;
 
-        const classes = classNames('tooltip', className, {
+        const classes = classNames('tooltip bdl-Tooltip', className, {
```

Comment:
There can only be one classPrefix in Tether, so I left it as the existing "tooltip" prefix: https://github.com/swfree/box-ui-elements/pull/2/files#diff-7752a80e54e2fb88494303ca60ee3044R256. Since the main conflict is with the root ".tooltip" class, and not others like ".tooltip-target-attached...", it seems fine to leave this as is for now. If we want to one day update the class prefix as well, we may have to introduce a breaking change to do so.